examples/peripheral_pedometer/src/step_calc.c

```diff
@@ -63,7 +63,7 @@ static pedometer_info_t result =
 
 #define ABS(a) (0 - (a)) > 0 ? (-(a)) : (a)
 
-#define DYNAMIC_PRECISION                 0
+#define DYNAMIC_PRECISION                 100
```

Comment:
Please revert to commit f27f687dd0673a345294b579f78da5abad734049, then use `board.c`.